tests/integration/targets/meraki_mr_ssid/meraki_ssid/tasks/main.yml

```diff
@@ -520,6 +522,7 @@
       net_name: TestNetSSID
       number: 1
       use_vlan_tagging: yes
+      visibile: true
```

Comment:
Looks like this is a spelling error.

Reply:
Thanks for the review, this should be fixed now